core/src/text/textBuffer.cpp

```diff
@@ -15,6 +15,22 @@ TextBuffer::TextBuffer(std::shared_ptr<VertexLayout> _vertexLayout)
 TextBuffer::~TextBuffer() {
 }
 
+std::vector<TextBuffer::WordBreak> TextBuffer::findWordBreaks(const std::string& _text) {
+    std::vector<WordBreak> breaks;
+
+    for (int i = 0, quad = 0; i < _text.size(); ++i) {
+        if (std::isspace(_text[i])) {
+            int lookAHead = i;
```

Comment:
very minor nitpick, the H in lookAHead is probably not meant to be capitalized :)